bundle/jsky.util/src/main/java/jsky/util/DateUtil.java

```diff
@@ -17,12 +17,12 @@
 public class DateUtil {
 
     /**
-     * Return a string with the UTC time in the format /mm/dd/yy hh:mm:ss,
+     * Return a string with the UTC time in the format yyyy-MMM-dd hh:mm:ss,
      * given the current UTC time in ms.
      */
     public static String formatUTC(long time) {
```

Comment:
When grepping for formatUTC I could find one usage of `formatUTC` in [`EdProgramHelper`](https://github.com/gemini-hlsw/ocs/blob/aa6c1b9131fb59aad82c067ffff2b906af523d38/bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/EdProgramHelper.scala#L39-L40).

Do you think that will be problematic?